test/activities/content/ContentModuleEntity.js

```diff
@@ -81,5 +81,11 @@ describe('ContentModuleEntity', () => {
 			}
 			expect(fetchMock.called()).to.be.true;
 		});
+
+		it('deletes module', async() => {
```

Comment:
i might suggest to add another layer of descriptions for these tests.. and with better names. this doest really give a lot of info:
example:
```
describe('Actions', ...)
  describe('delete', ...)
    it('calls fetch mock or whatever you expect'
```

Reply:
I don't personally know if that level of detail is really required here. I could change the test to `calls delete request` if this is an issue instead of adding another level of `describe` here. I know we aren't technically checking if a module is deleted but I think it gets the point across.